frontend/src/components/question-creation/question-elements/MultipleChoiceElement.tsx

```diff
@@ -11,40 +13,52 @@ import {
 } from "@chakra-ui/react";
 
 import { EditOutlineIcon } from "../../../assets/icons";
-import { MultipleChoiceData } from "../../../types/QuestionTypes";
+import {
+  MultipleChoiceData,
+  QuestionElementType,
+} from "../../../types/QuestionTypes";
 
 import EditMultipleChoiceModal from "./modals/multiple-choice/EditMultipleChoiceModal";
 
 interface MultipleChoiceElementProps {
   id: string;
   data: MultipleChoiceData;
+  type: QuestionElementType;
 }
 
 const MultipleChoiceElement = ({
```

Comment:
Could you update `MultipleChoiceElement` -> `MultiOptionElement` or something like that?